src/Aquifer.Public.API/Endpoints/Resources/Get/AvailableLanguages/Endpoint.cs

```diff
@@ -0,0 +1,49 @@
+﻿using Aquifer.Data;
+using FastEndpoints;
+using Microsoft.EntityFrameworkCore;
+
+namespace Aquifer.Public.API.Endpoints.Resources.Get.AvailableLanguages;
+
+public class Endpoint(AquiferDbContext dbContext) : Endpoint<Request, IEnumerable<Response>>
+{
+    public override void Configure()
+    {
+        Get("/resources/{ContentId}/available-languages");
+        Description(d => d.ProducesProblemFE(404));
+        Summary(s =>
+        {
+            s.Summary = "For a given resource content id, see in what other languages it is available.";
```

Comment:
Are we using the term `resource content id` in a public facing way? I was thinking we were saying `resource id`. It seems like the term `content id` and `resource id` are being used interchangeably but I'm wondering if it would be clearer if we aligned on just one term.

Reply:
The top of the document gives clarity that a resource is a piece of content as we're defining it in the public API. I was noticing what you mentioned, which is why I started using resource content id to be really explicit, since there's a concept of a resource type or a resource collection. Since we define a resource differently from how the broader community seems to (they would call a resource what we call a parent resource), I think it's important to keep the "content" part in there.

Reply:
Ok that's fair, I'd suggest maybe then we should do an audit of what we've got in the public API (not in this PR) and make sure we're being consistent with `resource content id`.